apps/stops/test/route_stop_test.exs

```diff
@@ -188,18 +188,20 @@ defmodule Stops.RouteStopTest do
       ])
     end
 
-    test "handles ferry routes with multiple shapes by returning the stops as-is" do
-      primary = %Shape{id: "primary"}
-      other = %Shape{id: "secondary"}
-      stops = make_stops(~w(Boat-Long Boat-Logan Boat-Hull Boat-Rowes Boat-Hingham))
-      route = %Route{id: "boat", type: 4}
-      actual = list_from_shapes([primary, other], stops, route, 1)
-
-      assert_stop_ids(
-        actual,
-        ~w(Boat-Long Boat-Logan Boat-Hull Boat-Rowes Boat-Hingham)
-      )
-    end
+    # Commenting out this test temporarily.
```

Comment:
Would it make sense to use the [skip tag](https://hexdocs.pm/ex_unit/ExUnit.Case.html#module-known-tags) here? (e.g. `@tag skip: "As of Summer 2020 the limited service does not include multiple shapes for this ferry."`)

Reply:
Oh that is better for sure, I didn't know about that.